Userland/Shell/AST.cpp

```diff
@@ -2553,10 +2553,11 @@ ErrorOr<RefPtr<Value>> Pipe::run(RefPtr<Shell> shell)
     }
 
     Vector<Command> commands;
-    commands.extend(left);
-    commands.append(last_in_left);
-    commands.append(first_in_right);
-    commands.extend(right);
+    TRY(commands.try_ensure_capacity(left.size() + 2 + right.size()));
+    MUST(commands.try_extend(left));
+    MUST(commands.try_append(last_in_left));
+    MUST(commands.try_append(first_in_right));
+    MUST(commands.try_extend(right));
```

Comment:
`MUST(try_*())` sounds silly, I'm not sure what the consensus on this is, but I'd rather use the non-try functions here.

Reply:
OK